src/routes/tickets/controllers/ticketController.ts

```diff
@@ -82,6 +83,11 @@ const validate = (method: IValidationMethods['method']): RequestHandler[] => {
         body('subject', 'Field subject failed validation').exists().isObject().notEmpty(),
       ];
     }
+    case 'getUserAuthoredTickets': {
+      return [
+        body('userId', 'Field userId failed validation').exists().isString().escape().isLength({ min: 0, max: 100 }),
```

Comment:
Shouldn't this be params?

Reply:
I'll use the check fn instead.

![bild](https://user-images.githubusercontent.com/16852656/146838601-5038c27a-9db0-4c85-ba53-09e1f94fa63d.png)